### DISCUSSION TITLE
Missing "Go to file" navigation menu on repo home

### DISCUSSION THREAD

**Discussion by creasty** [community/community#6037] [Mobile]
| Repo home | Other pages |
|---|---|
| ![IMG_1814](https://user-images.githubusercontent.com/1695538/136201787-c695b992-36de-441f-b962-5eb86b20864f.jpg) | ![IMG_1815](https://user-images.githubusercontent.com/1695538/136201792-ac750bd5-074d-422b-8a48-eb854eac61c7.jpg) |

| Repo home on desktop (for reference) |
|---|
| <img width="1032" alt="Screen Shot 2021-10-06 at 21 29 02" src="https://user-images.githubusercontent.com/1695538/136202225-a911d6c4-6d88-45eb-a2f4-8ae0ce570ee8.png"> |

**Comment by brianlovin** [comment]
Thanks for the suggestion @creasty — I agree! We don't have any work on the file browser prioritized right now, but I've added your +1 to our tracking issue to help us prioritize in the future.
